### PR TITLE
Change color of IN_Error, Cancelled, Failled and Killed

### PR DESCRIPTION
**Pull request by medou-boushab**

### CODE REVIEW

File: scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/view/grid/jobs/JobsListGrid.java [hunk 1]
```diff
@@ -177,13 +177,13 @@ protected String getJobStatusFieldColor(ListGridRecord record, int rowNum, int c
...
+                        return "color:#BE2D1C;font-weight:bold;" + base; // red color
                     case CANCELED:
-                        return "color:#939382;font-weight:bold;" + base; // Light Gray color
+                        return "color:#C50000;font-weight:bold;" + base; // Light Gray color
                     case FAILED:
-                        return "color:#FF8281;" + base; // Light Red color
+                        return "color:#850003;" + base; // Light Red color
                     case IN_ERROR:
-                        return "color:#6b6767;font-weight:bold;" + base; // Dark Gray color
+                        return "color:#E94504;font-weight:bold;" + base; // Dark Gray color
```
Comment (ShatalovYaroslav) [inline scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/view/grid/jobs/JobsListGrid.java]: Could you also change the comments of colors:
Dark Gray color -> red color

File: scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/view/grid/jobs/JobsListGrid.java [hunk 2]
```diff
@@ -177,13 +177,13 @@ protected String getJobStatusFieldColor(ListGridRecord record, int rowNum, int c
             try {
                 switch (getJobStatus(record)) {
                     case KILLED:
-                        return "color:#c50000;font-weight:bold;" + base; // red color
+                        return "color:#BE2D1C;font-weight:bold;" + base; // red color
                     case CANCELED:
-                        return "color:#939382;font-weight:bold;" + base; // Light Gray color
+                        return "color:#C50000;font-weight:bold;" + base; // Light Gray color
```
Comment (mboussaa) [inline scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/view/grid/jobs/JobsListGrid.java]: C50000 is not gray

File: scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/view/grid/jobs/JobsListGrid.java [hunk 3]
```diff
@@ -177,19 +177,19 @@ protected String getJobStatusFieldColor(ListGridRecord record, int rowNum, int c
...
-                        return "color:#FF8281;" + base; // Light Red color
+                        return "color:#850003;" + base; // dark Red color
                     case IN_ERROR:
-                        return "color:#6b6767;font-weight:bold;" + base; // Dark Gray color
+                        return "color:#E94504;font-weight:bold;" + base; // light orange color
                     case RUNNING:
-                        return "color:#1a8bba;font-weight:bold;" + base; //Dark  Blue color
+                        return "color:#2051F4;font-weight:bold;" + base; // Blue color
                     case PENDING:
                         return "color:#F0E130;font-weight:bold;" + base; // Yellow color: Dandelion color name
```
Comment (amouhoub) [inline scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/view/grid/jobs/JobsListGrid.java]: Pending for info is not bold

File: scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/view/grid/jobs/JobsListGrid.java [hunk 4]
```diff
@@ -177,19 +177,19 @@ protected String getJobStatusFieldColor(ListGridRecord record, int rowNum, int c
             try {
                 switch (getJobStatus(record)) {
                     case KILLED:
-                        return "color:#c50000;font-weight:bold;" + base; // red color
+                        return "color:#BE2D1C;font-weight:bold;" + base; // red color
                     case CANCELED:
-                        return "color:#939382;font-weight:bold;" + base; // Light Gray color
+                        return "color:#C50000;font-weight:bold;" + base; // red color
                     case FAILED:
-                        return "color:#FF8281;" + base; // Light Red color
```
Comment (amouhoub) [inline scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/view/grid/jobs/JobsListGrid.java]: failed should be bold (`font-weight:bold`)

**Pull request merged**